packages/contentful/src/delivery-api/poll-delivery-api.ts

```diff
@@ -0,0 +1,26 @@
+import { Entry as CDAEntry, EntrySkeletonType, FieldsType } from 'contentful';
+import retry from 'async-retry';
+
+export const pollContentfulDeliveryApi = async <
+  T extends EntrySkeletonType<FieldsType, string>,
+>(
+  fetchEntry: () => Promise<CDAEntry<T>>,
+  version: number,
```

Comment:
maybe not to confuse everyone even more let's make it accept a revision so it's clear that this method, as opposed to `pollContentfulGql`, uses a different field to poll?